CMakeLists.txt

```diff
@@ -693,6 +693,12 @@ else()
             set(UA_FILE_NSPUBSUB ${PROJECT_SOURCE_DIR}/tools/schema/Opc.Ua.NodeSet2.PubSubMinimal.xml)
         endif()
     endif()
+	if(UA_ENABLE_DA)
+		list(APPEND UA_FILE_DATATYPES ${PROJECT_SOURCE_DIR}/tools/schema/datatypes_dataaccess.txt)
+		if(UA_ENABLE_DA_INFORMATIONMODEL)
+			set(UA_FILE_NSDA ${PROJECT_SOURCE_DIR}/tools/schema/Opc.Ua.NodeSet2.Part8_Subset.xml)
```

Comment:
The variable UA_FILE_NSDA is not used in this PR.
Remove or use.

Reply:
I will remove this